taxonomy-editor-frontend/src/components/useFetch.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from "react";
+
+const useFetch = (url) => {    

```

Comment:
NOT relevant for this PR, but what about just using a library for all the fetching and stuff: [query](https://tanstack.com/query/v4)  does an excelent job on that and it is simple to use

Reply:
I think this kind of suggestion is useful @nobeeakon 
introducing right libraries up front to avoid reinventing the wheel latter on ;-)

(but this is a difficult equilibrium to get unless you are experimented, so cool to see suggestions on that).